src/applications/disability-benefits/all-claims/pages/evidenceTypes.js

```diff
@@ -0,0 +1,79 @@
+import { validateBooleanGroup } from 'us-forms-system/lib/js/validation';
+import { validateBooleanIfEvidence } from '../validations';
+import get from '../../../../platform/utilities/data/get';
+
+import {
+  evidenceTypeHelp,
+  noEvidenceDescription
+} from '../content/evidenceTypes';
+
+export const uiSchema = {
+  'view:hasEvidence': {
+    'ui:title': 'Do you have any evidence that you’d like to submit with your claim?',
+    'ui:widget': 'yesNo'
+  },
+  'view:hasEvidenceFollowUp': {
+    'ui:options': {
+      expandUnder: 'view:hasEvidence'
+    },
+    'ui:required': (formData) => get('view:hasEvidence', formData, false),
+    'view:selectableEvidenceTypes': {
+      'ui:title': 'What type of evidence do you want to submit with your claim?',
+      'ui:options': { showFieldLabel: true },
+      'ui:validations': [{
+        validator: validateBooleanIfEvidence,
+        options: { wrappedValidator: validateBooleanGroup }
+      }],
+      'ui:errorMessages': {
+        atLeastOne: 'Please select at least one type of supporting evidence'
+      },
+      'ui:required': (formData) => get('view:hasEvidence', formData, false),
+      'view:hasVAMedicalRecords': { 'ui:title': 'VA medical records' },
+      'view:hasPrivateMedicalRecords': { 'ui:title': 'Private medical records' },
+      'view:hasOtherEvidence': { 'ui:title': 'Supporting (lay) statements or other evidence' }
+    },
+    'view:evidenceTypeHelp': {
+      'ui:title': ' ',
+      'ui:description': evidenceTypeHelp
+    }
+  },
+  'view:noEvidenceFollowUp': {
+    'ui:title': ' ',
```

Comment:
I'm sure this is needed, but I forget why. Do we normally add a title of `' '` when we don't want to give an element a title, but want it to create a label so we can show validation errors on it? Is there another reason for this?

Ideally, we can express why we do this so we can file a ticket in USFS...so we don't _have_ to do this anymore.